src/dep.rs

```diff
@@ -292,14 +293,16 @@ impl<'a> Visit for DependencyCollector<'a> {
       }
       Expr::Tpl(tpl) => {
         if tpl.quasis.len() == 1 && tpl.exprs.is_empty() {
-          DynamicArgument::String(tpl.quasis[0].raw.clone())
+          DynamicArgument::String(
+            tpl.quasis[0].cooked.as_ref().unwrap().clone(),
+          )
         } else {
           let mut parts =
             Vec::with_capacity(tpl.quasis.len() + tpl.exprs.len());
           for i in 0..tpl.quasis.len() {
-            if tpl.quasis[i].raw.len() > 0 {
-              parts
-                .push(DynamicTemplatePart::String(tpl.quasis[i].raw.clone()));
+            let cooked = tpl.quasis[i].cooked.as_ref().unwrap();
+            if cooked.len() > 0 {
+              parts.push(DynamicTemplatePart::String(cooked.clone()));
```

Comment:
@dsherret PTAL - the previous code was incorrect. Raw includes escape sequences in raw form.